client/components/mma/accountoverview/AccountOverview.tsx

```diff
@@ -156,8 +157,14 @@ const AccountOverviewPage = () => {
 			isSpecificProductType(productDetail, PRODUCT_TYPES.digipack),
 		);
 
+	const hasNonServiceableCountry = nonServiceableCountries.includes(
+		allActiveProductDetails.find(isProduct)?.billingCountry as string,
+	);
+
 	const isEligibleToSwitch =
-		!maybeFirstPaymentFailure && !hasDigiSubAndContribution;
+		!maybeFirstPaymentFailure &&
```

Comment:
I love the "maybe" in this variable name. I wonder if we could make the flag more deterministic? (Not a required change)

Reply:
I love the maybe!